test/images/jessie-dnsutils/BASEIMAGE

```diff
@@ -1,4 +1,4 @@
-amd64=debian:jessie
```

Comment:
given that this image is named `jessie-dnsutils`, I wonder if it should stay as just `debian:jessie`? or if we should rename it?